src/utils/compileSlots.ts

```diff
@@ -15,10 +15,13 @@ export function processSlot(source = '', Vue = vue) {
     `<SlotWrapper v-bind="$attrs">${template}</SlotWrapper>`,
     {
       mode: 'function',
-      prefixIdentifiers: true
+      prefixIdentifiers: __BROWSER__
```

Comment:
I think you may need to update tsconfig (or somewhere else?) to make this compile.